### PR TITLE
[TTPReq] Remaining Education Steps for US and UK

### LINKED ISSUE #14321: [TTPReq] Education: How to accept contactless card With Tap to Pay on iPhone
Designs & Copy: pdfdoF-5KF-p2
Flows: pdfdoF-5GF-p2
Depends on: https://github.com/woocommerce/woocommerce-ios/issues/14159

## Education Step Screen Functionality

If it's not already done, I suggest creating a reusable full-screen scroll view to fit as much educational content as needed. Support text blocks, or numbered text blocks as we have in 'Set up Tap to Pay'.

## How to accept contactless card With Tap to Pay on iPhone

- Use the visual assets from [Marketing Guidelines](https://developer.apple.com/tap-to-pay/marketing-guidelines/)
- Use the latest bezels
- Use localized images for all the supported TTP markets

## Designs

Image

### PR DESCRIPTION
**Pull request by staskus**
Closes: #14321, #14322, #14165
Depends on: https://github.com/woocommerce/woocommerce-ios/pull/14596
Designs: pdfdoF-5KF-p2

## Description

Configure all the steps for TTP education:
- Accept contactless payments with only an iPhone.
- How to accept contactless card with Tap to Pay on iPhone.
- How to accept Apple Pay and other digital wallets with Tap to Pay on iPhone.
- How to handle PIN entry for a card. (UK-only, tested, works)
- How to accept Apple Pay and other digital wallets with Tap to Pay on iPhone. (UK-only, uses an image provided by Woo designers since Apple doesn't provide it)

Images taken from https://developer.apple.com/tap-to-pay/marketing-guidelines/#visual-assets.

## Steps to reproduce

Prerequisites:
- Enable `tapToPayEducation` feature flag in `DefaultFeatureFlagService`
- TTP compatible iPhone
- Have US and UK located stores ready

1. Menu
2. Payments
3. About Tap to Pay
4. Confirm that all the described steps are shown (3 in the US, 4 in the UK)
5. Confirm that the image shows currency based on the market ($ - US store, £ - UK store)
6. Check the text and images, provide any observations
7. Check dark/light mode, larger text, accessibility 

## Testing information

Tested on iPhone 14 Pro 17.7.

### US

https://github.com/user-attachments/assets/b58d7bd1-5cf2-467b-b7b9-4f9ba0c293a8

### UK

https://github.com/user-attachments/assets/b653651e-710a-4b3e-a675-7b9a9ccf69af

---
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:

- [ ] The PR is small and has a clear, single focus, or a valid explanation is provided in the description. If needed, please request to split it into smaller PRs.
- [ ] Ensure Adequate Unit Test Coverage: The changes are reasonably covered by unit tests or an explanation is provided in the PR description.
- [ ] Manual Testing: The author listed all the tests they ran, including smoke tests when needed (e.g., for refactorings). The reviewer confirmed that the PR works as expected on all devices (phone/tablet) and no regressions are added.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/PaymentSettingsFlowHint.swift [hunk 1]
```diff
@@ -9,14 +9,16 @@ struct PaymentSettingsFlowHint: View {
...
+                .background(Color(.init(light: .systemGray6, dark: .darkGray)))
                 .clipShape(Circle())
             Text(text)
                 .font(.callout)
                 .padding(.leading, 16)
             Spacer()
         }
         .padding(.horizontal, 8)
+        .accessibilityElement()
+        .accessibilityLabel("\(number). \(text)")
```
Comment (jaclync) [inline WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/PaymentSettingsFlowHint.swift:21]: nice accessibility support 💯

File: WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/Tap to Pay Education/TapToPayEducationStepsFactory.swift [hunk 2]
```diff
@@ -57,4 +75,104 @@ private enum Localization {
...
+            comment: "Title for the 'PIN entry' Tap to Pay merchant education step"
+        )
+
+        static let description = NSLocalizedString(
+            "tapToPay.education.step.pin.description",
+            value: "Customers are prompted to enter their card PIN under specific circumstances with Tap to Pay on iPhone.\n\n" +
+                   "For customers needing visual or other assistance, accessibility options are accessed by selecting " +
+                   "‘Accessibility Options’ on the PIN screen. Audible instructions guide customers to draw their PIN on the " +
+                   "screen or tap the screen to indicate each digit - tapping once for 1, twice for 2, and so on. " +
+                   "To submit their PIN, they simply swipe right with two fingers.",
```
Comment (jaclync) [inline WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/Tap to Pay Education/TapToPayEducationStepsFactory.swift:154]: oh nice, PIN code is supported in TTP - TIL. Even better with accessibility support.

**Review by jaclync — APPROVED** [review]
Looks great in both US and UK stores! :shipit:

**Pull request merged**